test/Base.t.sol

```diff
@@ -247,6 +247,7 @@ abstract contract Base_Test is DeployOptimized, Events, Merkle, V2CoreAssertions
 
     function computeMerkleStreamerLLAddress(
```

Comment:
should we add one more function without asset param?

```solidity 
    function computeMerkleStreamerLLAddress(
        address admin,
        bytes32 merkleRoot,
        uint40 expiration
    )
        internal
        returns (address)
    {
     computeMerkleStreamerLLAddress(admin, address(dai), merkleRoot, expiration);
    }
```

Reply:
Why? In your suggestions above, you recommended combining two functions into one using input parameters. Then why do you suggest to add one more function in this case?

Reply:
to have a greater flexibility 

similar to what we have in `Defaults`. one function has the asset one has not:

https://github.com/sablier-labs/v2-periphery/blob/1804e530aabd274106655d14e4da2c7a1e39e431/test/utils/Defaults.sol#L181-L197

Reply:
OK.